roles/core/log_client/readme.rst

```diff
@@ -34,13 +34,61 @@ Log client verbosity defaults to info, it can be one of the following (defined i
 | debug    | debug-level messages             |
 +----------+----------------------------------+
 
+
+You can split the configuration into several files, using **log_client_configuration_files**. 
+This variable can also be used to override the default rsyslog configuration.
+It uses variables:
+- **name**: the configuration file name
+- **content**: the content of your configuration file, it can be single line or multiline
+- **path**: the directory containing your configuration file, it defaults to /etc/rsyslog.d if it is not specified
+
+For example:
+
+.. code-block:: yaml
+
+  log_client_configuration_files:
+    - name: local-rules.conf
+      path: /etc/rsyslog.d
```

Comment:
I would remove the path since this is the default. An example is available below with /etc.